src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private final double base;
+    private final double lateralSide;
+
+    IsoscelesTrapezoid(String color, double base, double lateralSide) {
+        super(color);
+
+        this.base = base;
+        this.lateralSide = lateralSide;
+    }
+
+    @Override
+    public double getArea() {
+        double firstLeg = base / 2;
+        double height = Math.pow(lateralSide, 2) - Math.pow(firstLeg, 2);
+
+        return base * height * 0.5;
```

Comment:
The formula for the area of an isosceles trapezoid is incorrect. The correct formula is `(base1 + base2) / 2 * height`, where `base1` and `base2` are the lengths of the two parallel sides. You need to include the other base of the trapezoid in your calculation.

Reply:
Oh okay